mlflow/types/chat.py

```diff
@@ -159,7 +159,9 @@ class BaseRequestPayload(BaseModel):
 
     temperature: float = Field(0.0, ge=0, le=2)
     n: int = Field(1, ge=1)
-    stop: Optional[list[str]] = Field(None, min_items=1)
+    stop: Optional[list[str]] = (
+        Field(None, min_length=1) if IS_PYDANTIC_V2_OR_NEWER else Field(None, min_items=1)
```

Comment:
Does passing kwargs to Field work? e.g. `kwargs = {"min_length": 1} if IS_PYDANTIC_V2_OR_NEWER else {"min_items": 1}, Field(..., **kwargs)` so that you don't need to change all lines with the if check

Reply:
We could do this, but I think the current one is more readable.
```python
Field(None, **({"min_length": 1} if IS_PYDANTIC_V2_OR_NEWER else {"min_items": 1})) 
```